agent/utils.py

```diff
@@ -98,6 +100,51 @@ def filter_description(description: str) -> str:
     return description
 
 
+def _sort_dict(dictionary: dict[str, Any] | list[Any]) -> dict[str, Any] | list[Any]:
```

Comment:
If this function will be used only for sorting locations, i would name it sort_location
```suggestion
def _sort_location(location: dict[str, Any] | list[Any]) -> dict[str, Any] | list[Any]:
```